dvisvgm does not work with gs-9.27 because GS_PDF_ProcSet was removed in
gs-9.27.

--- psdefs.cpp.orig	Mon Mar 11 03:21:29 2019
+++ psdefs.cpp	Thu May 30 08:06:00 2019
@@ -107,8 +107,8 @@
 "dmode sysexec<</Normal 0/Compatible 0/Multiply 1/Screen 2/Overlay 3/SoftLight "
 "4/HardLight 5/ColorDodge 6/ColorBurn 7/Darken 8/Lighten 9/Difference 10/Exclus"
 "ion 11/Hue 12/Saturation 13/Color 14/Luminosity 15/CompatibleOverprint 16>>exc"
-"h get 1(setblendmode)prcmd}def/@pdfpagecount{GS_PDF_ProcSet begin pdfdict begi"
-"n(r)file pdfopen begin pdfpagecount currentdict pdfclose end end end}def/@pdfp"
-"agebox{GS_PDF_ProcSet begin pdfdict begin(r)file pdfopen begin dup dup 1 lt ex"
-"ch pdfpagecount gt or{pop}{pdfgetpage/MediaBox pget pop aload pop}ifelse curre"
-"ntdict pdfclose end end end}def DELAYBIND{.bindnow}if ";
+"h get 1(setblendmode)prcmd}def/@pdfpagecount{(r)file runpdfbe"
+"gin pdfpagecount runpdfend}def/@pdfp"
+"agebox{(r)file runpdfbegin dup dup 1 lt ex"
+"ch pdfpagecount gt or{pop}{pdfgetpage/MediaBox pget pop aload pop}ifelse runpd"
+"fend}def DELAYBIND{.bindnow}if ";
